CHANGELOG.md

```diff
@@ -23,6 +24,167 @@
+- New Admin API endpoint `/tags/` for listing entities by tag: `/tags/example_tag`
+
+##### PDK
+
+- New PDK function: `kong.client.get_protocol` for obtaining the protocol
+  in use during the current request
+  [#4307](https://github.com/Kong/kong/pull/4307)
+- New PDK function: `kong.nginx.get_subsystem`, so plugins can detect whether
+  they are running on the HTTP or Stream subsystem
+  [#4358](https://github.com/Kong/kong/pull/4358)
+
+##### Plugins
+
+- :fireworks: Support for ACL **authenticated groups**, so that authentication plugins
+  that use a 3rd party (other than Kong) to store credentials can benefit
+  from using a central ACL plugin to do authorization for them
+  [#4013](https://github.com/Kong/kong/pull/4013)
+- The Kubernetes Sidecar Injection plugin is now bundled into Kong for a smoother K8s experience
+  [#4304](https://github.com/Kong/kong/pull/4304)
+- aws-lambda: includes AWS China region.
+  Thanks [@wubins](https://github.com/wubins) for the patch!
+  [#4176](https://github.com/Kong/kong/pull/4176)
+
+### Changes
+
+##### Dependencies
+
+- The required OpenResty version is still 1.13.6.2, but for a full feature set
+  including stream routing and Service Mesh abilities with mutual TLS, Kong's
+  [openresty-patches](https://github.com/kong/openresty-patches) must be
+  applied (those patches are already bundled with our official distribution
+  packages). The openresty-patches bundled was updated in Kong 1.1.0 to include
+  the `stream_realip_module` as well.
+  [#4163](https://github.com/Kong/kong/pull/4163)
+  Kong in HTTP(S) Gateway scenarios does not require these patches.
+- Service Mesh abilities require at least OpenSSL version 1.1.1. In our
+  official distribution packages, OpenSSL has been bumped to 1.1.1b.
+  [#4345](https://github.com/Kong/kong/pull/4345)
+
+### Fixes
+
+##### Core
+
+- Resolve hostnames properly during initialization of Cassandra contact points
+  [#4296](https://github.com/Kong/kong/pull/4296),
+  [#4378](https://github.com/Kong/kong/pull/4378)
+- Fix health checks for Targets that need two-level DNS resolution
+  (e.g. SRV → A → IP) [#4386](https://github.com/Kong/kong/pull/4386)
+- Fix serialization of map types in the Cassandra backend
+  [#4383](https://github.com/Kong/kong/pull/4383)
+- Fix target cleanup and cascade-delete for Targets
+  [#4319](https://github.com/Kong/kong/pull/4319)
+- Avoid crash when failing to obtain list of Upstreams
+  [#4301](https://github.com/Kong/kong/pull/4301)
+
+##### Admin API
+
+- Proper support for `PUT /{entities}/{entity}/plugins/{plugin}`
+  [#4288](https://github.com/Kong/kong/pull/4288)
+- Fix Admin API inferencing of map types using form-encoded (#4368)
+  [#4368](https://github.com/Kong/kong/pull/4368)
+
+##### Plugins
+
+- basic-auth, ldap-auth, key-auth, jwt, hmac-auth: fixed
+  status code for unauthorized requests: they now return 403
+  instead of 401
```

Comment:
The other way around: they now return 401 instead of 403 (403 being Forbidden). I like to always specify "HTTP 401", which provides some more context and helps the reader picture HTTP status code (vs exit codes, or who knows what).